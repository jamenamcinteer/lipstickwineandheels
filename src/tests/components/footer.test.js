import React from 'react'
import { shallow } from 'enzyme'
import Footer from '../../components/footer'

test('should render footer component correctly', () => {
  const wrapper = shallow(<Footer />)
  expect(wrapper).toMatchSnapshot()
})
