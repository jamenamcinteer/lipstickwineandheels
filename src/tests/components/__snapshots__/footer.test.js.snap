// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render footer component correctly 1`] = `
<footer
  className="footerContainer"
>
  <div
    className="footerWrapper"
  >
    <p>
      <GatsbyLink
        to="/page/style-guide/"
      >
        Style Guide
      </GatsbyLink>
    </p>
    <p>
      © 2018-2019. All Rights Reserved.
    </p>
    <p>
      Made with ❤️ and ☕ in Austin, Texas.
    </p>
    <p
      className="footerBuilt"
    >
      Built by Jamena McInteer using GatsbyJS, ReactJS, GraphQL, and Contentful
    </p>
  </div>
</footer>
`;
